i6_models/util/mask.py

```diff
@@ -0,0 +1,16 @@
+import torch
+
+
+def mask_tensor(tensor: torch.Tensor, seq_len: torch.Tensor) -> torch.Tensor:
```

Comment:
I'm not sure about the name. I find it a bit too generic. And also ambiguous. (Does it mask a tensor? No, actually, it returns a mask tensor. For what? Based on what?)

In TF, the corresponding function is called `tf.sequence_mask`. Why not just use the same name here? So all people who have used TF at some point in the past (almost everyone) knows what this is about.

Reply:
tf.sequence_mask has a different signature, and we can also add such a function. This is just not what I wanted to have here.

Reply:
For the name, yes this can be more verbose if needed.